### ISSUE TITLE
Problem with package: Pillow

### ISSUE THREAD

**Issue by missionfloyd** [piwheels/packages#357]
### Package name

pillow

### Package version

all

### PyPI URL

https://pypi.org/project/pillow/

### piwheels URL

https://www.piwheels.org/project/pillow/

### Python version

- [X] Python 3.7
- [X] Python 3.9

### I am the maintainer

- [ ] Yes

### More information

The pillow wheel doesn't seem to be compiled with libraqm support, required for text features such as ligatures and non-left-to-right text. Running this on a Raspberry Pi returns `False` unless the user compiles it themselves.
```python
>>> from PIL import features
>>> features.check_feature("raqm")
```
See also:
https://github.com/python-pillow/Pillow/issues/6023
https://github.com/python-pillow/Pillow/issues/7121

**Comment by MichaIng** [comment]
To solve this, the builder would require the `libraqm-dev` package.

**Comment by bennuttall** [comment]
Is there any downside to this? I remember from early days having issues if we built packages with extra features, which required more dependencies from users.

**Comment by MichaIng** [comment]
Indeed, it would require `libraqm0`, which pulls `libfreetype6`, `libfribidi0`, and `libharfbuzz0b` as dependencies. ~8 MiB overall, if none was installed already.
